src/api/app/models/staging_workflow.rb

```diff
@@ -32,6 +32,33 @@ def ignored_requests
     BsRequest.none # TODO: define this method
   end
 
+  def to_xml
+    builder = Nokogiri::XML::Builder.new
+    staging_workflow_attributes = { project: project.name,
+                                    who: User.current_login,
```

Comment:
this who is strange - who saved the workflow last is part of the _history of _project, no need to repeat it within the xml. if nothing changed, the xml should stay the same - who didn't change it, is of no interet :)

Reply:
same for when actually - also in _history

Reply:
I will remove it